.pre-commit-config.yaml

```diff
@@ -0,0 +1,53 @@
+---
+default_language_version:
+  python: python3.9
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.3.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+        exclude: ^.*\.(md|rst)$
+      - id: debug-statements
+      - id: mixed-line-ending
+        args: [--fix=lf]
+      - id: check-merge-conflict
+
+  - repo: https://github.com/jorisroovers/gitlint
+    rev: v0.14.0
+    hooks:
+      - id: gitlint
+
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.25.0
+    hooks:
+      - id: yamllint
+
+  - repo: https://github.com/PyCQA/pydocstyle
+    rev: 5.1.1
+    hooks:
+      - id: pydocstyle
+        files: ^src/.*
+
+  - repo: https://github.com/ambv/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        exclude: ^docs/.*
+        types: [python]
+        args: [--safe, --quiet, --line-length, "88"]
+        require_serial: true
+
+  - repo: https://github.com/asottile/seed-isort-config
```

Comment:
Should it be here? This PR is requiring us to use isort 5 (given the removal of --recursive).  The repo is archived and on the web page https://github.com/asottile-archive/seed-isort-config  they say: "this is no longer needed as of isort>=5".

Reply:
Removed